src/assets/sass/protocol/components/_split.scss

```diff
@@ -17,16 +17,16 @@
 
     @media #{$mq-md} {
         padding: get-theme('v-grid-md') 0;
+
+        &.mzp-t-split-nospace {
```

Comment:
Moved this into the media query so it still gets default spacing on small screens (since media can't pop out on mobile anyway).